stitching/stitcher.py

```diff
@@ -105,7 +114,7 @@ def stitch(self, img_names):
         seam_masks = self.find_seam_masks(imgs, corners, masks)
 
         imgs = self.resize_final_resolution()
-        imgs, masks, corners, sizes = self.warp_final_resolution(imgs, cameras)
+        imgs, masks, corners, sizes = self.warp_final_resolution(imgs, cameras, feature_masks)
```

Comment:
we don't need the feature masks in final resolution

Reply:
Discussion in [#55](https://github.com/lukasalexanderweber/stitching/pull/55#issuecomment-1457960716)